cent/context_controller.py

```diff
@@ -0,0 +1,32 @@
+from typing import Optional, TYPE_CHECKING, Any
+
+from pydantic import BaseModel, PrivateAttr
+
+if TYPE_CHECKING:
+    from cent.client import Client
+
+
+class ClientContextController(BaseModel):
```

Comment:
Not obvious - what is the purpose of this class?

Reply:
Implemented this class for the future, if you need to call some methods based on the returned object

Reply:
Could you show example? still do not understand the intent